pkg/reconciler/clusteringress/resources/gateway.go

```diff
@@ -45,10 +45,10 @@ var placeholderServer = v1alpha3.Server{
 }
 
 // GetServers gets the `Servers` from `Gateway` that belongs to the given ClusterIngress.
-func GetServers(gateway *v1alpha3.Gateway, ci *v1alpha1.ClusterIngress) []v1alpha3.Server {
+func GetServers(gateway *v1alpha3.Gateway, name string) []v1alpha3.Server {
```

Comment:
I'm fine with sharing things in `resources`, but would rather have the shared code live in it's final location, so that the ultimate removal is a simple `git rm -rf ./pkg/reconciler/clusteringress`.  As-is, we'll have to carefully move the things we need first.

Reply:
The main reason is that diffs show up better this way for reviews.   We will do git mv the files and remove the clusteringress specifics.   The end result is the same.

Reply:
Shared modules in `resources` have been from `clusteringress` to `ingress`.